client-lib/src/main/java/com/ibm/ws/repository/resources/internal/RepositoryResourceImpl.java

```diff
@@ -929,7 +930,7 @@ public RepositoryResourceMatchingData createMatchingData() {
 
         // connect to massive and find that sample
```

Comment:
Could we remove or correct this comment whilst your in this area?  It doesn't seem to match the code.